src/lang-promql/complete/hybrid.ts

```diff
@@ -281,4 +257,53 @@ export class HybridComplete implements CompleteStrategy {
           );
         });
   }
+
+  private arrayToCompletionResult(
+    data: AutoCompleteNode[],
+    from: number,
+    to: number,
+    context: AutocompleteContext,
+    state: EditorState,
+    includeSnippet = false
+  ): CompletionResult {
+    const text = state.sliceDoc(from, to);
+    const options: Completion[] = [];
+    const fuzzyOption = { pre: this.fuzzyPre, post: this.fuzzyPost };
+
+    for (const completionList of data) {
+      for (const label of completionList.labels) {
+        if (context.filterType === FilterType.Fuzzy) {
+          const result = fuzzy.filter(text, [escape(label)], fuzzyOption);
```

Comment:
Btw., if you're passing just one element at a time, we are not using half of `fuzzy.filter()`'s functionality, because it also does sorting by score. You can probably just copy this bit right here:

https://github.com/mattyork/fuzzy/blob/master/lib/fuzzy.js#L120-L129

...with the thing in the `if` block just being a different variant of what you already have here.

Reply:
sorting is perfomed by CMN after (using the score). So it's fine to do that from my point of view

Reply:
I'm just using fuzzy lib here to calculate the score and to inject the html when configured

Reply:
Yeah, exactly. Since we are doing sorting externally, we could just extract the tiny bit from `fuzzy.filter()` that we actually need here.

Reply:
I.e. just use `fuzzy.match()` directly instead of `fuzzy.filter()`.

Reply:
aaah ok right. Sorry, it was bit confused in my head.

Reply:
oh yeah it's so much better. Thanks a lot for the tips !

Reply:
here we go, fixed !